recipe/test_opendap1_url.patch

```diff
@@ -0,0 +1,41 @@
+--- pynio-1.5.0_beta20160623.orig/test/test-opendap2.py	2016-06-23 19:16:59.000000000 -0300
++++ pynio-1.5.0_beta20160623/test/test-opendap2.py	1969-12-31 21:00:00.000000000 -0300
+@@ -1,38 +0,0 @@
+-# 
+-# This is a test for OPeNDAP. It will only work on systems in which NCL
+-# has OPenDAP capabilities built in.
+-#
+-# I'm using what looks like a test opendap server. See:
+-#
+-# http://test.opendap.org/opendap/data/nc/contents.html
+-#
+-import sys, Nio
+-
+-if not Nio.__formats__['opendap']:
+-    print '==========================='
+-    print 'Optional format OPeNDAP is not enabled in this version of PyNIO'
+-    print '==========================='
+-    sys.exit()
+-
+-url      = "http://test.opendap.org/opendap/data/nc/"
```

Comment:
Many OPeNDAP URLs are down lately. I am not up to speed with the news but I did hear something about a security risk and a preference for ERDDAP instead.